src/pages/Components/RatingStars/code.md

```diff
@@ -3,39 +3,198 @@ title: 'Code'
 
 ```html
-<div class="class">Label</div>
+<div class="mc-stars-input">
+  <input type="radio" name="rating" value="0" id="no-rate" class="sr-only mc-stars-input__radio" checked aria-label="No opinion given" />
+
+  <input type="radio" name="rating" value="1" id="ratingstars1" class="sr-only mc-stars-input__radio">
+  <label for="ratingstars1" class="mc-stars-input__label">
+    <span class="sr-only mc-stars-input__text">Very bad</span>
+    <svg class="mc-stars-input__icon"><use xlink:href="#starFullS" /></svg>
+  </label>
+
+  <input type="radio" name="rating" value="2" id="ratingstars2" class="sr-only mc-stars-input__radio">
+  <label for="ratingstars2" class="mc-stars-input__label">
+    <span class="sr-only mc-stars-input__text">Bad</span>
+    <svg class="mc-stars-input__icon"><use xlink:href="#starFullS" /></svg>
+  </label>
+
+  <input type="radio" name="rating" value="3" id="ratingstars3" class="sr-only mc-stars-input__radio">
+  <label for="ratingstars3" class="mc-stars-input__label">
+    <span class="sr-only mc-stars-input__text">Medium</span>
+    <svg class="mc-stars-input__icon"><use xlink:href="#starFullS" /></svg>
+  </label>
+
+  <input type="radio" name="rating" value="4" id="ratingstars4" class="sr-only mc-stars-input__radio">
+  <label for="ratingstars4" class="mc-stars-input__label">
+    <span class="sr-only mc-stars-input__text">Good</span>
+    <svg class="mc-stars-input__icon"><use xlink:href="#starFullS" /></svg>
+  </label>
+
+  <input type="radio" name="rating" value="5" id="ratingstars5" class="sr-only mc-stars-input__radio">
+  <label for="ratingstars5" class="mc-stars-input__label">
+    <span class="sr-only mc-stars-input__text">Excellent</span>
+    <svg class="mc-stars-input__icon"><use xlink:href="#starFullS" /></svg>
+  </label>
+</div>
 ```
 
-<preview path="src/pages/Components/RatingStars/previews/RatingStars"></preview>
+<preview path="src/pages/Components/RatingStars/previews/stars-input-basic"></preview>
+
+### Variations
+
+#### Available sizes
 
-## Variations
+The rating stars are available in 4 sizes.
 
-use the same variations in the same order as the index page
+To change the size of the stars, simply apply one of the following modifier **to the main container** :
 
-## Behaviors
+- **small** : `mc-stars-input--s`
+- **medium** : this is the default size so you don't need to add a modifier class
+- **large** : `mc-stars-input--l`;
+- **extralarge** : `mc-stars-input--xl`;
+
+```html
+<div class="mc-stars-input mc-stars-input--s">...</div>
+
+<div class="mc-stars-input mc-stars-input--l">...</div>
+```
+
+<preview path="src/pages/Components/RatingStars/previews/stars-input-all-sizes"></preview>
+
+### Behaviors
 
 use the same Behaviors in the same order as the index page
 
-## Responsive behaviors
+### Responsive behaviors
```

Comment:
no content ? remove, but should we be able to change size depending on the viewport ?